frontend/src/lib/components/apps/components/table/AppTable.svelte

```diff
@@ -133,7 +129,7 @@
 
 			<div class="overflow-x-auto flex-1 w-full">
 				<table class="relative w-full border-b border-b-gray-200">
-					<thead class="sticky top-0 z-50 bg-gray-50 text-left">
```

Comment:
@adam-kov Had to remove this, because the anchor + move button were rendering under the table header

Reply:
This will cause the action buttons to render above the table header